ccf_transaction_processor/Makefile

```diff
@@ -21,6 +21,10 @@ ifndef PDO_INSTALL_ROOT
 $(error Incomplete configuration, PDO_INSTALL_ROOT is not defined)
 endif
 
+ifndef PDO_LEDGER_KEY_ROOT
+$(error Incomplete configuration, PDO_LEDGER_KEY_ROOT is not defined)
+endif
+
 ifndef SGX_MODE
```

Comment:
Does it make sense to source common-config.sh as part of make itself, so that defaults are never undefined?

Reply:
not really. the configuration variables are necessary for both run and build.